firmware/ec/src/post/post.c

```diff
@@ -192,6 +192,7 @@ static void post_move_to_next_subsystem()
  **    RETURN TYPE     : None
  **
  *****************************************************************************/
+#if 0
```

Comment:
Did you block comment these out because of warnings that they are declared and unused? I think this will lead to confusion in the future, can you either add a clear comment as to why it's block commented out or (my preferred way) add a GCC pragma surrounding these functions to ignore the warning for just these, along with a comment saying these are implemented and may be needed in the future, but aren't used now?

Reply:
Same reason as line 102 in eth_sw.c.  We can either add a pragma around unused functions, or add the prototype to a header file.  Which is preferred?

Reply:
Added comment to clarify intention.